husky_extensions/ckeditor-extension.js

```diff
@@ -23,7 +23,7 @@
         }
     });
 
-    define(['ckeditor', 'jqueryAdapter'], function() {
+    define(['underscore', 'ckeditor', 'jqueryAdapter'], function(_) {
```

Comment:
Why do we need `'ckeditor'` and `'jqueryAdapter'` here, although it is not passed to the function?